portfolio/src/main/webapp/script.js

```diff
@@ -109,11 +91,18 @@ function secretPage(){
     //The button to access the secret page is hidden until the user enters a correct answer 
     let secret = document.getElementById('secret-button');
```

Comment:
`secret` can be marked `const` instead of `let` since the value of `secret` is never reassigned